### DISCUSSION TITLE
My website is not showing after installation your FreshRSS

### DISCUSSION THREAD

**Discussion by satheeshsoft** [FreshRSS/FreshRSS#3778] [Q&A] [answered]
Hi,
My name is Satheesh. My website is https://softcleaner.in its a software related website. Its using cPanel (i think linux). Through cPanel i use wordpress. Today 18-8-2021 i install your RSS feed from wordpress. After installing i gone to my website main page ie https://softcleaner.in. The whole contents in the main page as well as other pages too gone/not showing. (pages are available on the wordpress but not visible now).

The main page is showing rss feed login page instead of my description there.
So i though to uninstall the FreshRSS and clicked to uninstall. It shown some directory also needed to delete fore ever. I feared and unchecked those directories and done uninstall. It uninstalled.

Then also the webpages are not showing anything.

So i thought to again re-install and uninstall those directories as per your software saying. Again installed and uninstalled without unchecking those directories during the uninstall time.

Now also same nothing is showing in the webpages.
What can i do?
Regards,
Satheesh

My website was working properly before. (I already written in issues)
Which folders need to remove? can you specific.
**
Before it was showing https://softcleaner.in  now it is showing https://softcleaner.in/p/i/

**Comment by Frenzie** [comment, marked as answer]
I'm not sure how cPanel would install the script, but I'm afraid it's only tangentially related to the repo here on GH.

It sounds like the uninstaller script left behind an index.php (and possibly a few other files) from FreshRSS. You'd have to restore the one from Wordpress. Within a cPanel context, presumably some kind of reinstall for Wordpress would be the best/easiest option.

**Reply by satheeshsoft** [reply]
It installed from the wordpress and the wordpress taken from cPanel dashboard. From wordpress i installed and everything gone unexpected, basically i or any developer didn't expect. So trying to a paid restore the entire public_html files from the hostgator_gibo service provider. I think the entire directory some files changed. You need to check your program in same situations, so that you can find out where the problem persists and to succeed. Its an error of every software but need to fix.

**Reply by Frenzie** [reply]
As I said, no one here has anything to do with the cPanel dashboard. There is no way to protect against something or someone overwriting one file with another completely outside the program.

Almost every PHP-based program will feature a file named index.php in its root directory, so you should normally run them from different (sub)directories.

**Reply by satheeshsoft** [reply]
I think your are correct, i downloaded the same root directory because it didn't show another own directory by default. So that i thought no need to do from my side, it does automatically. I think the index file changed.
cPanel is normal only, its a good dashboard. I think you if you can create the sub folder by default its easy for like me who unaware about web programming.

**Reply by satheeshsoft** [reply]
website is working now with the help of hostgator.
